releasenotes.md

```diff
@@ -6,6 +6,10 @@ GeoIP2 .NET API Release Notes
 
 * `AverageIncome` and `EstimatePopulation` were added to the `Location`
```

Comment:
@2shortplanks, I noticed that this API used `estimated_population`. This is the old name, right? Do we need to update it to `population_density`? Any other changes?